src/openhab.cpp

```diff
@@ -413,8 +375,13 @@ void OpenHAB::processItems(const QJsonDocument& result, bool first) {
             }
         }
         if ((_myEntities.count() - countFound) > 0) {
-            m_notifications->add(true,
-                                 "openHAB - entities missing : " + QString::number((_myEntities.count() - countFound)));
+            m_notifications->add(
+                        true, "Could not load : " + QString::number((_myEntities.count() - countFound))
+                        + "openHAB items", tr("Reconnect"), [](QObject* param) {
+                Integration* i = qobject_cast<Integration*>(param);
+                i->connect();
```

Comment:
Does it make sense to offer the user to reconnect to OH?
If I understand this correctly, then this is only a notification message that items are missing, but the connection itself is still valid.

Reply:
That's true. My idea was to give the user the change to reconnect but it#s just an information. Changed it back.